### DISCUSSION TITLE
I accidentally left the organization, is it possible to recover it?

### DISCUSSION THREAD

**Discussion by beingtyson** [vercel/community#6792] [Account Issue]
### Question

Question
Recently, I left the organization after sending an invitation email to another account, can I rejoin the organization?

For your information, the account that received the invitation cannot join after the invitation period has passed.

**Comment by amyegan** [comment]
Hey @beingtyson. If you you left a Vercel team or GitHub organization, you should contact the owner and ask that they add you again

**Reply by beingtyson** [reply]
@amyegan I understand that a new user can be invited as an owner. However, my current situation is that there is no one in the organization. I invited a new user as an owner and left, but since they did not accept the invitation and it expired, there's effectively no one left in the team. The service is still operational, and I want to re-register a payment method to resume normal operations.

**Reply by amyegan** [reply]
The system would prevent you from leaving a team without another confirmed owner. I was able to find what I believe to be the team you left, and it does have a current owner with a relatively new account. 

The new owner should see the team as an option in the [team picker](https://vercel.com/guides/how-do-i-select-a-team-on-vercel) near the top of the dashboard. After selecting the team, they should be able to follow the steps for [adding a team member](https://vercel.com/docs/accounts/team-members-and-roles/managing-team-members#adding-team-members-and-assigning-roles) to invite you back to the team. 

If the new owner is having difficulty logging in, please ask them to email [registration@vercel.com](mailto:registration@vercel.com) to reach the support team that can verify their account information to help them log in.
